website/docs/language/settings/backends/configuration.mdx

```diff
@@ -42,6 +42,7 @@ There are some important limitations on backend configuration:
 
 - A configuration can only provide one backend block.
 - A backend block cannot refer to named values (like input variables, locals, or data source attributes).
+- The values declared within backend blocks cannot be [referenced](https://www.terraform.io/language/expressions/references#references-to-resource-attributes) elsewhere in the configuration.
```

Comment:
```suggestion
- You cannot [reference](https://www.terraform.io/language/expressions/references#references-to-resource-attributes) values declared within backend block elsewhere in the configuration.
```

Suggest changing "The values declared within backend blocks cannot be referenced" to "You cannot reference values declared in backend blocks" to avoid a passive voice style issue.

Reply:
Excellent point. I'll commit this change immediately. Thanks!